build.sbt

```diff
@@ -10,21 +10,17 @@ scmInfo in ThisBuild := Some(ScmInfo(
   url("https://github.com/slamdata/sbt-quasar-datasource"),
   "scm:git@github.com:slamdata/sbt-quasar-datasource.git"))
 
-// Include to also publish a project's tests
-lazy val publishTestsSettings = Seq(
-  publishArtifact in (Test, packageBin) := true)
-
 lazy val root = project
   .in(file("."))
-  .settings(noPublishSettings)
-  .aggregate(core)
-  .enablePlugins(AutomateHeaderPlugin)
-
-lazy val core = project
-  .in(file("core"))
-  .settings(name := "sbt-quasar-datasource")
   .settings(
-    /*
-    libraryDependencies += ...
-     */)
+    name := "sbt-quasar-datasource",
+    sbtPlugin := true,
+    sbtVersion := "1.1.6")
```

Comment:
We should probably just use 1.2.6, which I believe is the latest. SBT uses semver so it *should* be backwards compatible, at least for core things.